### DISCUSSION TITLE
Missing documentation for -- numchannels

### DISCUSSION THREAD

**Discussion by gene96817** [jamulussoftware/jamulus#2015] [Documentation]
Somewhere along the way https://jamulus.io/wiki/Command-Line-Options has information about --numchannels.
Could someone also check to see if other options have been dropped?
Thanks.

**Comment by chrisrimple** [comment]
Maybe related to #1837 ?

**Comment by gilgongo** [comment]
It's in the docs for the next release: https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/wiki/en/Running-a-Server.md Not sure why it got omitted for a while. Next release of the docs should be a in a few weeks once all translations are in.

**Reply by gene96817** [reply]
Thanks.
